src/Support/ModelScope.php

```diff
@@ -77,6 +84,26 @@ public function with(array $relations, ?string $modelClass = null): self
         return $this;
     }
 
+    /**
+     * Set the callback that should have an opportunity to modify the database query.
+     *
+     * @param callable $callback
```

Comment:
Please remove `@param` and `@return`, these don't add any value when strict types are used